### PR TITLE
Adjusted tooltip style to business

### PR DESCRIPTION
**Pull request by RitzAnthony**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,4 @@ projects/sbb-esta/angular-business/select
 projects/sbb-esta/angular-business/textarea
 projects/sbb-esta/angular-business/time-input
 projects/sbb-esta/angular-business/usermenu
+projects/sbb-esta/angular-business/tooltip
```
Comment (kyubisation) [inline .gitignore]: Why was this added?

File: projects/sbb-esta/angular-business/tooltip/src/tooltip/tooltip.component.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ import {
 
 @Component({
   selector: 'sbb-tooltip',
-  templateUrl: '../../../../angular-public/tooltip/src/tooltip/tooltip.component.html',
+  templateUrl: './tooltip.component.html',
```
Comment (kyubisation) [inline projects/sbb-esta/angular-business/tooltip/src/tooltip/tooltip.component.ts]: Where is the template?

File: projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip-base.scss [hunk 3]
```diff
@@ -0,0 +1,238 @@
+@import '../../../../angular-core/styles/common/variants';
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip-base.scss]: Why was this extracted to its own file?
The intention going forward is to remove these separate styles folders and move the sass code into the *.component.scss files.

File: projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip-variables.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
+@import '../../../../angular-core/styles/common/colors';
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip-variables.scss]: See comment in _tooltip-base.scss

File: projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip.scss [hunk 5]
```diff
@@ -0,0 +1,2 @@
+@import './tooltip-variables';
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/tooltip/src/styles/_tooltip.scss]: See comment in _tooltip-base.scss

File: projects/sbb-esta/angular-public/tooltip/src/tooltip/tooltip.component.scss [hunk 6]
```diff
@@ -1,251 +1 @@
...
-
-  .sbb-tooltip-icon {
-    @include tooltipIcon();
-    float: right;
-    position: absolute;
-    top: toPx($tooltipIconsSize);
-    right: toPx($tooltipIconsSize);
-  }
-}
+@import '../styles/tooltip';
```
Comment (kyubisation) [inline projects/sbb-esta/angular-public/tooltip/src/tooltip/tooltip.component.scss]: See comment in _tooltip-base.scss

**Comment by efux** [comment]
Hello @kyubisation and @RitzAnthony. We are planning on using the Tooltip component in our projects. Is there still some progress for this issue/PR?

**Comment by kyubisation** [comment]
@efux This PR is out of date. We bumped priority of #90 and #225 and will try to provide business tooltips in the next two weeks, based on the existing PRs.

**Comment by jeripeierSBB** [comment]
@efux @RitzAnthony With version 9.2.0 business styling of tooltip is now available.
